.github/dependabot.yml

```diff
@@ -9,6 +9,11 @@ updates:
     labels:
       - "ok-to-test"
       - "release-note-none"
+    groups:
+      kubernetes:
+        patterns:
+          - "k8s.io/*"
+          - "sigs.k8s.io/*"
```

Comment:
I don't think sigs should be part of the same group.

Reply:
I agree. Pushed up a fix for that.